pkg/webhook/webhook.go

```diff
@@ -247,10 +246,19 @@ func (e *Validator) handlePush(ctx context.Context, event *github.PushEvent) (*g
 	log.Infof("%+v\n%+v", resp, resp.Files)
 	var files []string
 	for _, file := range resp.Files {
-		if ok, err := filepath.Match(".github/chainguard/*.sts.yaml", file.GetFilename()); err == nil && ok {
+		if strings.HasPrefix(file.GetFilename(), ".github/chainguard/") {
 			files = append(files, file.GetFilename())
 		}
 	}
+	var nonSTSFiles []string
+	for _, f := range files {
+		if !strings.HasSuffix(f, ".sts.yaml") {
+			nonSTSFiles = append(nonSTSFiles, f)
+		}
+	}
+	if len(nonSTSFiles) > 0 {
+		return e.handleNonSTSFiles(ctx, client, owner, repo, sha, nonSTSFiles)
+	}
```

Comment:
If there are non-sts files we ignore all other validation?

Reply:
That's true, yeah. I didn't want to reach too far into the "real" validation method to account for the case where there's both kinds of errors, but I can if you'd prefer that.

With this change as it is, if someone had an invalid `.sts.yaml` and a non-sts-`.yaml` they'd get a failure for the latter, fix it, and get another failure for the former, which is kind of sub-optimal, but at least it doesn't let bad stuff sneak through.